lib/cdk_node.star

```diff
@@ -71,7 +76,12 @@ def get_cdk_node_ports(args):
             wait=aggregator_wait,
         ),
         "rpc": PortSpec(
-            args["zkevm_cdk_node_port"],
+            args["cdk_node_rpc_port"],
+            application_protocol="http",
+            wait=None,
+        ),
+        "rest": PortSpec(
+            args["cdk_node_rest_api_port"],
```

Comment:
do we need to add this in case of aggregator?

Reply:
I would add it, just in case, because previously we were adding the `rpc` port, and the bridge service as well was listening on it.